make-bower-build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="master"
+
+# Run only test cases if this is not a release
+if [ -z "$TRAVIS_TAG" ]; then
+    echo "Skipping deploy; just doing a build and running test cases"
+    npm test
+    exit 0
+fi
+
+npm test
+
+# Get the deploy key by using Travis's stored variables to decrypt deploy_key.enc
+ENCRYPTED_KEY_VAR="encrypted_${ENCRYPTION_LABEL}_key"
+ENCRYPTED_IV_VAR="encrypted_${ENCRYPTION_LABEL}_iv"
+ENCRYPTED_KEY=${!ENCRYPTED_KEY_VAR}
+ENCRYPTED_IV=${!ENCRYPTED_IV_VAR}
+openssl aes-256-cbc -K $ENCRYPTED_KEY -iv $ENCRYPTED_IV -in id_rsa_nginfinite.enc -out deploy_key -d
+chmod 600 deploy_key
+eval `ssh-agent -s`
+ssh-add deploy_key
+
+BOWER_REPO='git@github.com:sroze/ngInfiniteScroll-bower.git'
```

Comment:
`This should go to git@github.com:ng-infinite-scroll/ng-infinite-scroll-bower.git`